lib/sdf-v1-routes-actions/BUCK

```diff
@@ -0,0 +1,22 @@
+load("@prelude-si//:macros.bzl", "rust_library")
+
+rust_library(
+    name = "sdf-v1-routes-actions",
+    deps = [
+        "//lib/dal:dal",
+        "//lib/sdf-core:sdf-core",
+        "//lib/sdf-extract:sdf-extract",
+        "//lib/si-events-rs:si-events",
+        "//lib/si-layer-cache:si-layer-cache",
+        "//lib/telemetry-rs:telemetry",
+
```

Comment:
I wouldn't mind if we didn't add the empty lines here--the idea was simply to lexical sort this list of deps, not necessarily to delineate between "our deps" and "other deps". In the Buck2 instance, these happen to sort apart. I may be able to help with a future lint/format rule, but just find it harder to keep this sorting alphabetical when there are blank lines in the middle.